go.mod

```diff
@@ -3,7 +3,7 @@ module chainguard.dev/melange
 go 1.23.4
 
 require (
-	chainguard.dev/apko v0.25.3
+	chainguard.dev/apko v0.25.4-0.20250312233451-b6bac08fe6d0
```

Comment:
We'll want an official Apko release before considering merging.

Reply:
Addressed in [`6406bc6` (#1836)](https://github.com/chainguard-dev/melange/pull/1836/commits/6406bc6e7823815aceb47754b98e6b338145b8ba).